src/components/views/DownloadModModal.vue

```diff
@@ -98,17 +97,17 @@ let assignId = 0;
 
         static allVersions: [number, DownloadProgress][] = [];
 
-        get activeGame(): Game {
-            return this.$store.state.activeGame;
-        }
-
         get ignoreCache(): boolean {
             const settings = this.$store.getters['settings'];
             return settings.getContext().global.ignoreCache;
         }
 
-        setSelectedVersion(newSelectedVersion: string) {
-            this.selectedVersion = newSelectedVersion;
+        selectedVersionChanged(event: Event) {
+            if (!(event.target instanceof HTMLSelectElement)) {
```

Comment:
Moot point now, but it would be nicer if this was done in the child the component that emits, as that's where the markup and thus the event target is defined. The parent component is only interested in the version string, so we should pass only that and hide the internal implementation of the child component.